docs/CMakeLists.txt

```
@@ -74,13 +74,19 @@ set(CMAKE_MODULE_PATH "${PROJECT_SOURCE_DIR}/cmake/modules" ${CMAKE_MODULE_PATH}
 find_package(Sphinx REQUIRED)
 
 set(DOCS_BUILDER html)
+if (${CMAKE_BUILD_TYPE} STREQUAL "Debug")
+    set(SHOW_TODOS "-Dtodo_include_todos=1")
+else()
+    set(SHOW_TODOS "-Dtodo_include_todos=0")
+endif()
 
 # Generate the sphinx documentation
 add_custom_target(Sphinx ALL
     COMMAND
     ${SPHINX_EXECUTABLE} -b ${DOCS_BUILDER}
     # Tell Breathe where to find the Doxygen output
     -D breathe_projects.fastdds-statistics-backend=${DOXYGEN_OUTPUT_XML_DIR}
+    ${SHOW_TODOS}
```

As `todo_include_todos` is `false` by default I think that using generator expressions here could be better.
```suggestion
    $<$<STREQUAL:"${CMAKE_BUILD_TYPE}","Debug">:-Dtodo_include_todos=1>
```

---

Niiiiiice